CHANGELOG.unreleased.md

```diff
@@ -6,6 +6,7 @@
 ### Enhancements
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
+> [Backup] Go back to previous page instead of going to the overview when saving a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```

Comment:
```suggestion

- [Backup] Go back to previous page instead of going to the overview when saving a backup job: keeps the user's current filters and page (PR [#5913](https://github.com/vatesfr/xen-orchestra/pull/5913))
```